briefcase/django.py

```diff
@@ -78,13 +82,45 @@ def install_extras(self):
         # Install additional elements required for Django
         print(" * Installing extras...")
         print("   - Installing NPM requirements...")
-
-        npm = shutil.which("npm")
-        subprocess.Popen([npm, "install"], cwd=os.path.abspath(self.dir)).wait()
-
+        node_attributes = {
+            'darwin': {
+                'url':'https://nodejs.org/dist/v6.10.3/node-v6.10.3.pkg',
+                'command_prefix': 'open '
+            },
+            'linux' : '',
+            'win32' : {
+                'url' :'https://nodejs.org/dist/v6.10.3/node-v6.10.3-x86.msi',
+                'command_prefix': 'msiexec /i '
+            }
+        }
+        system_platform = sys.platform
+        node = shutil.which('node')
+        npm  = shutil.which("npm")
+        #workaround in case no node version is found
+        node_version = 'No version'
+        try:
+            node_version_unicode = subprocess.check_output([node, '--version'])
+            node_version = node_version_unicode.decode('utf-8')
+            print('Node version %s detected' % node_version)
+        except:
+            node_version = ''
+        if node and node_version[1] == '6':
+            subprocess.Popen(['npm', 'install'], cwd=os.path.abspath(self.dir)).wait()
+        else:
+            if not node or node_version[1] != '6':
+                err_message = ('ERROR: Cannot run with the current version of Node and Npm, please \n'
+                               'unnistall the current version and install Node version 6.x\n'
+                               'Installation cancelled.'
+                )
+            raise RuntimeError(err_message)
         print("   - Building Webpack assets...")
         subprocess.Popen([npm, "run", "build"], cwd=os.path.abspath(self.dir)).wait()
 
+    def django_migrate(self):
```

Comment:
There's an argument that we should migrate for the user automatically, can you open an issue on briefcase to discuss it, and remove this code? Thanks!